packages/contracts/contracts/L2/L2ToL1MessagePasser.sol

```diff
@@ -62,21 +72,20 @@ contract Withdrawer {
     function initiateWithdrawal(
         address _target,
         uint256 _gasLimit,
```

Comment:
too large gas limit.

Reply:
This is safe, downstream there is a `require(gasleft() > _gasLimit + buffer)` 

I lean towards not changing any of the gas limits here as the only one that needs to be a uint64 is the one used for deposits. Unless there is strong preference for it to be a uint64 everywhere just for sanity reasons, which I would understand too

Reply:
I think that it could be fine. I just flagged each instance without too much thought. It's just that any gas limit that serializes to go / actual execution needs to fit in 64 bits (and probably way fewer) otherwise the deposit gets stuck

Reply:
Thoughts @smartcontracts?